ksql-engine/src/test/java/io/confluent/ksql/security/KsqlAuthorizationImplTest.java

```diff
@@ -310,26 +310,6 @@ public void shouldCreateAsSelectExistingTopicWithWritePermissionsAllowed() {
     // Above command should not throw any exception
   }
 
-  @Test
-  public void shouldThrowWhenCreateAsSelectExistingStreamWithoutWritePermissionsDenied() {
-    // Given:
-    givenTopicPermissions(TOPIC_1, Collections.singleton(AclOperation.READ));
-    givenTopicPermissions(TOPIC_2, Collections.singleton(AclOperation.READ));
-    final Statement statement = givenStatement(String.format(
-        "CREATE STREAM %s AS SELECT * FROM %s;", STREAM_TOPIC_2, STREAM_TOPIC_1)
-    );
-
-    // Then:
-    expectedException.expect(KsqlTopicAuthorizationException.class);
-    expectedException.expectMessage(String.format(
-        "Authorization denied to Write on topic(s): [%s]", TOPIC_2.name()
-    ));
-
-
-    // When:
-    authorizationValidator.checkAuthorization(serviceContext, metaStore, statement);
-  }
-
```

Comment:
I removed this test because I removed the Write check for CSAS/CTAS on the authorization implementation. 

The reason to remove it was because it was not doing any real check on the sink topic. It was done in a Sandboxed environment only. I will write another PR to fix this issue.